.github/workflows/lint.yml

```diff
@@ -0,0 +1,123 @@
+name: Lint
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  lint:
```

Comment:
Ha, it looks like the new YML settings are a lot more verbose than before. But it'd be good to have inline linting and type-checking messages, yes!